packages/lou_bank_ui/lou_bank_gallery/lib/theme/color_screen.dart

```diff
@@ -0,0 +1,159 @@
+import 'package:flutter/material.dart';
+import 'package:lou_bank_ui/lou_bank_ui.dart';
+
+class ColorScreen extends StatelessWidget {
+  const ColorScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Colors')),
+      body: ListView(children: <Widget>[
+        ListTile(
+          title: const Text('white = Colors.white'),
+          trailing: SizedBox(
+            width: 42.0,
+            height: 42.0,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                  color: LouColors.white,
+                  border: Border.all(color: Colors.black)),
+            ),
+          ),
+        ),
+        ListTile(
+          title: const Text('yellow = Color(0xffF2FE8D)'),
+          trailing: SizedBox(
+            width: 42.0,
+            height: 42.0,
+            child: DecoratedBox(
+              decoration: BoxDecoration(
+                  color: LouColors.yellow,
+                  border: Border.all(color: Colors.black)),
+            ),
+          ),
+        ),
+        ListTile(
```

Comment:
List tile is being repeated. MAke a widget out of it that will take things that change as a parameter.